app/views/NewExploreDeep/index.tsx

```diff
@@ -138,234 +208,595 @@ const representationOptions: Option[] = [
     },
 ];
 
+interface TimeOption {
+    key: string;
+    label: string;
+    startDate: number;
+    endDate: number;
+}
+
+const yearOptions: TimeOption[] = [
+    {
+        key: '2022',
+        label: '2022',
+        startDate: 1640974500000,
+        endDate: 1672424100000,
+    },
+    {
+        key: '2021',
+        label: '2021',
+        startDate: 1609438500000,
+        endDate: 1640888100000,
+    },
+    {
+        key: '2020',
+        label: '2020',
+        startDate: 1577816100000,
+        endDate: 1609352100000,
+    },
+    {
+        key: '2019',
+        label: '2019',
+        startDate: 1546280100000,
+        endDate: 1577729700000,
+    },
+    {
+        key: '2018',
+        label: '2018',
+        startDate: 1514744100000,
+        endDate: 1546193700000,
+    },
+];
+
+const yearKeySelector = (year: TimeOption) => year.key;
+const yearLabelSelector = (year: TimeOption) => year.label;
+
+const lastYearDateTime = new Date(`${lastYearStartDate}T00:00`).getTime();
+const todaysDateTime = new Date(`${todaysDate}T00:00`).getTime();
+const deepStartDateTime = new Date(`${DEEP_START_DATE}T00:00`).getTime();
```

Comment:
Let's use utility function

Reply:
Is there any?

Reply:
Let's create one. I think here is one anyway called resolveTime or something that was copied from another project.